openapi/components/schemas/ReadyToPay/Methods/ReadyToPayKlarnaMethod.yaml

```
@@ -0,0 +1,19 @@
+type: object
+title: Klarna
+required:
+  - method
+properties:
+  method:
+    type: string
+    description: The payment method.
+    enum:
+      - Klarna
+  feature:
+    type: object
+    nullable: true
+    description: |
+      The specific feature (for example, digital wallet or a processor) of this method. If method doesn't have any features it will be null.
```

nitpick: Avoid using contractions ("isn't", "don't", "shouldn’t" ... ) and parenthesis, where possible

> Parenthesis "( )": Avoid using parenthesis in text when possible. Readers interpret this in different ways and it disrupts the flow of a sentence or paragraph. 

```suggestion
      The specific feature of this method. For example, a digital wallet or a processor. If the method does not have any features, this value will be null.
```